package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/ezolenko/rollup-plugin-typescript2",
   "author": "@ezolenko",
   "scripts": {
-    "prebuild": "rimraf dist/*",
+    "prebuild": "rimraf dist/* && rexreplace '(requredTypescriptRange = .)[^\"]+' '€1+require(process.cwd()+\"/package.json\").peerDependencies.typescript' -j src/*.ts",
```

Comment:
This is better done in `rollup.config.base.js`, there is already `rollup-plugin-re` invocation in there. See how `$RPT2_VERSION` is treated.